src/components/map/BiodivInformation.vue

```diff
@@ -32,12 +33,15 @@
     },
 
     computed: {
-      description() {
+      descriptionParagraphs() {
         let result = this.data.description || '';
 
-        // very simple html > text interpreter : only html entities
-        result = result.replace(/&eacute;/g, 'é');
-        result = result.replace(/&egrave;/g, 'è');
+        // keep line breaks for better readability
+        result = result.replace(/<br ?\/?>/g, '%%MYNL%%');
+        // decode html entities
+        result = utils.decodeHtmlEntities(result);
+        // restore initial line breaks
+        result = result.split('%%MYNL%%');
```

Comment:
you're lucky we are not doing any type checking... but ok, it works.
for readability, I would prefer to use another variable for the returned array